frontend/src/pages/SignUp/index.tsx

```diff
@@ -98,69 +72,94 @@ const SignUp: React.FC = () => {
           <Stack
             component="form"
             direction="column"
-            spacing={1.5}
```

Comment:
Should we keep this? I feel that it looks better with this spacing (relative to the form field heights) if we include the `size="small"` attributes on the form fields (see my comments below!)